src/lib/sql/sql_translator.cpp

```diff
@@ -1574,8 +1575,16 @@ std::shared_ptr<AbstractExpression> SQLTranslator::_translate_hsql_expr(
               aggregate_expression = std::make_shared<AggregateExpression>(
                   aggregate_function, _translate_hsql_expr(*expr.exprList->front(), sql_identifier_resolver));
             }
+          } break;
         }
 
+        // Check that the aggregate can be calculated on the given expression
+        const auto aggregate_data_type = aggregate_expression->data_type();
+        AssertInput(aggregate_data_type != DataType::Null,
+                    std::string{"Invalid aggregate "} + aggregate_expression->as_column_name() +
```

Comment:
Why do you wrap "Invalid aggregate " with `std::string{..}`? For " for input data type " you did not do it

Reply:
You need an std::string to start using `operator+`. At least it was that way a while ago.

Reply:
Interesting. I assumed today's compilers were clever enough to figure this out themselves.